sources/packages/api/src/route-controllers/models/esdc.shared.dto.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Shared DTO for EDSC file processing API response.
+ * TODO: Other esdc file processing services must eventually use this DTO.
+ */
+export class ESDCFileResponseAPIOutDTO {
```

Comment:
There is already a shared DTO for this one `ESDCFileResponseDTO`.
The name is not following the conventions because it is old, but it is a shared one. Can we use it?

Reply:
Ah, I thought it was not shared. I believe I need to move the controller class as well inside esdc-integration folder.

Reply:
I would suggest not creating a new controller. IMO the operation is related to the e-Cert files and it can stay in the existing e-Cert controller.